packages/redux/src/entities/selectors/products.ts

```diff
@@ -96,7 +96,7 @@ export const getProductLabelsByPriority = createSelector(
   [
     (state: StoreState, productId: ProductEntity['id']) =>
       getProduct(state, productId),
-    (state: StoreState, productId: ProductEntity['id'], sortOrder: string) =>
+    (state: StoreState, productId: ProductEntity['id'], sortOrder = 'asc') =>
```

Comment:
Is it possible to type the sortOrder parameter? From the code I can only see it has a default value 'asc' but I cannot know which values it can accept.

Reply:
Sure nice catch!